src/pages/blog-posts.js

```diff
@@ -13,12 +26,11 @@ class BlogPosts extends React.Component {
       <div>
         {posts.map(post => (
           <div>
-            <BlogPost key={post.node.slug} post={post.node} />
-            <Link to={`/${post.node.slug}`}>{post.node.title}</Link>
-            <br />
+            <SimplifiedBlogPost key={post.node.slug} post={post.node} />
```

Comment:
same here. Could this be a functional component?

Reply:
This refactor didn't work.  Having posts inside the functional component threw an error, but so did moving it outside the component.  Not sure what the proper syntax would be to get this working as an FC.  Could we look at it Friday if you have time :)

Reply:
of course!